src/main/config.cpp

```diff
@@ -256,6 +256,14 @@ void DBConfig::CheckLock(const string &name) {
 	throw InvalidInputException("Cannot change configuration option \"%s\" - the configuration has been locked", name);
 }
 
+bool FileExistsSafe(FileSystem &fs, const char *path) noexcept {
```

Comment:
Would we prefer a dedicated fs method for FileExistsSafe? Happy to do that - just didn't want to add an additional FileSystem interface unless required.